doc/python/histograms.md

```diff
@@ -208,6 +208,24 @@ fig = px.histogram(df, x="total_bill", color="sex", marginal="rug", # can be `bo
 fig.show()
 ```
 
+### Adding bar text
```

Comment:
so this example actually uses `px.bar` but in this page we want to focus on usage of `px.histogram` instead.

Reply:
I've updated it to use an example with `px.histogram`.